test/TensorFlow/retain_release.swift

```diff
@@ -13,7 +13,7 @@ public func test3Adds(x: Tensor<Int32>, y: Tensor<Int32>, z: Tensor<Int32>) {
 }
 
 // CHECK-LABEL: --- TFPartition Host Result: {{.*}}test3Adds{{.*}}
-// CHECK: sil shared @{{.*}}test3Adds{{.*}} : $@convention(thin)
+// CHECK: @{{.*}}test3Adds{{.*}} : $@convention(thin)
 
```

Comment:
a heads-up that i've seen piper behave differently in this regard. one option is to replace with {{.*}}

Reply:
FileCheck doesn't require that a `CHECK` matches the entire line, so it should already work no matter what stuff appears in front.